docs/v0.2.0/user/tls-passthrough.md

```diff
@@ -51,7 +51,7 @@ kubectl apply -f https://raw.githubusercontent.com/envoyproxy/gateway/v0.2.0/exa
 Patch the Gateway from the Quickstart guide to include a TLS listener that listens on port `6443` and is configured for TLS mode Passthrough:
 
 ```console
-$ kubectl patch gateway eg --type=json --patch '[{
+kubectl patch gateway eg --type=json --patch '[{
```

Comment:
same comment as above

Reply:
Sorry, I forgot to unify the code block language